### ISSUE TITLE
Declaration Order Question

### ISSUE THREAD

**Issue by mturnwall** [brigade/scss-lint#694]
I've always told my team that the declaration order for rulesets should be extend, mixin, and then regular declaration blocks

```scss
.my-class {
    @extend %myExtend;
    @include myMixin;
    font-size: 20px;
}
```

I want to use the Declaration Order rule in scss-lint, but I'm running into a problem. In my modules I like to declare my responsive rules at the bottom of the module.

```scss
.miniAlert {

    &--read {}

    // nested rulesets

    @include breakpoint($breakpoint-mobile) {}
}
```

I group it like this so that the responsive rules for the module are grouped with it and contained within the module's namespace. Visually you can see you are overriding rules. This throws a warning when I have DeclarationOrder enabled. Is there anyway around this?

**Comment by sds** [comment]
Hey @mturnwall,

We currently have an open issue (#494) for adding support for configurable orders to the `DeclarationOrder` linter. Would love a pull request!
